frontend/packages/dev-console/integration-tests/support/pages/app.ts

```diff
@@ -0,0 +1,163 @@
+      detailsPage.titleShouldContain('Monitoring');
+      break;
+    }
+    case devNavigationMenu.Builds: {
+      cy.byLegacyTestID('build-header').click();
+      detailsPage.titleShouldContain('Build Configs');
+      break;
+    }
+    case devNavigationMenu.Pipelines: {
+      cy.byLegacyTestID('pipeline-header').click();
+      detailsPage.titleShouldContain('Pipelines');
+      break;
+    }
+    case devNavigationMenu.Search: {
+      cy.byLegacyTestID('search-header').click();
+      detailsPage.titleShouldContain('Search');
+      break;
+    }
+    case devNavigationMenu.Helm: {
+      cy.byLegacyTestID('helm-releases-header').click();
+      detailsPage.titleShouldContain('Helm Releases');
+      break;
+    }
+    case devNavigationMenu.Project: {
+      cy.byLegacyTestID('project-details-header').click();
+      break;
+    }
+    case devNavigationMenu.ConfigMaps: {
+      cy.get('#ConfigMap').click();
+      detailsPage.titleShouldContain('Config Maps');
+      break;
+    }
+    case devNavigationMenu.Secrets: {
+      cy.get('#Secret').click();
+      detailsPage.titleShouldContain('Secrets');
+      break;
+    }
+    default: {
+      throw new Error('Option is not available');
+    }
+  }
+};
+
+export const projectNameSpace = {
+  selectCreateProjectOption: () => {
+    cy.byLegacyTestID('namespace-bar-dropdown')
+      .find('button')
+      .eq(0)
+      .click();
+    cy.byTestDropDownMenu('#CREATE_RESOURCE_ACTION#').click();
+  },
+
+  enterProjectName: (projectName: string) => {
+    cy.get('form[name="form"]').should('be.visible');
+    cy.get('#input-name').type(projectName);
+  },
+
+  clickCreateButton: () => {
+    cy.get('#confirm-action').click();
+  },
+
+  createNewProject: (projectName: string) => {
+    cy.byLegacyTestID("namespace-bar-dropdown")
+    .find('button')
+    .eq(0)
+    .click();
+    cy.byLegacyTestID('dropdown-text-filter').type(projectName);
+    cy.get('[role="listbox"]', {timeout:5000}).then(($el) => {
+      if ($el.find('li[role="option"]').length === 0) {
+        cy.byTestDropDownMenu('#CREATE_RESOURCE_ACTION#').click();
+        projectNameSpace.enterProjectName(projectName);
+        projectNameSpace.clickCreateButton();
+      }
+    });
+  },
+  selectOrCreateProject: (projectName: string) => {
+    cy.byLegacyTestID("namespace-bar-dropdown")
+      .find('button')
+      .eq(0)
+      .click();
+    cy.byLegacyTestID('dropdown-text-filter').type(projectName);
+    cy.get('[role="listbox"]').then(($el) => {
+      if ($el.find('li[role="option"]').length === 0) {
+        cy.byTestDropDownMenu('#CREATE_RESOURCE_ACTION#').click();
+        projectNameSpace.enterProjectName(projectName);
+        projectNameSpace.clickCreateButton();
+      }
+      else {
+        cy.get(`[id="${projectName}-link"]`).click();
+      }
+    });
+  },
+
+  selectProject: (projectName: string) => {
+    cy.byLegacyTestID("namespace-bar-dropdown")
+      .find('button')
+      .eq(0)
```

Comment:
Hi, what does `.eq(0)` do here?

Reply:
In dev-console, there are 2 instances with cy.byLegacyTestID("namespace-bar-dropdown").find('button'), so to identify the first instance, eq(0) is used

Reply:
ah..thanks for the info. :-)